### PR TITLE
Update ref for IsolatedStorage

### PR DESCRIPTION
**Pull request by JeremyKuhne**
Remove items that were removed from NetStandard 2.0 and add
upstream interfaces that are now implemented.

@weshaggard, @ericstj

### CODE REVIEW

File: src/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.cs [hunk 1]
```diff
@@ -114,29 +114,26 @@ public partial class IsolatedStorageFileStream : System.IO.FileStream
         public override bool CanWrite { get { throw null; } }
         // https://github.com/dotnet/corefx/issues/11127
         // [System.ObsoleteAttribute("This property has been deprecated.  Please use IsolatedStorageFileStream's SafeFileHandle property instead.  http://go.microsoft.com/fwlink/?linkid=14202")]
```
Comment (weshaggard) [inline src/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.cs]: keep the obsolete attribute in the ref.

File: src/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.cs [hunk 2]
```diff
@@ -152,20 +149,6 @@ public enum IsolatedStorageScope
         Roaming = 8,
         User = 1,
     }
-
-    public enum IsolatedStorageSecurityOptions
```
Comment (weshaggard) [inline src/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.cs]: Were these in the implementation? If so you can remove them there as well. If they weren't then I would expect a baseline file update.

File: src/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.cs (hunk 1 above)
Comment (JeremyKuhne) [inline src/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.cs]: I can't without updating FileStream. I'm going to update the issue mentioned to do so.

File: src/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.cs (hunk 2 above)
Comment (JeremyKuhne) [inline src/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.cs]: Ok, I've removed them. I was originally going to leave the trivial implementations, but I don't have a great argument for keeping them.

**Comment by JeremyKuhne** [comment]
@dotnet-bot test Innerloop CentOS7.1 Debug Build and Test

**Comment by JeremyKuhne** [comment]
@dotnet-bot Innerloop Windows_NT Release Build and Test

**Comment by JeremyKuhne** [comment]
@dotnet-bot test Innerloop CentOS7.1 Debug Build and Test

**Comment by JeremyKuhne** [comment]
@dotnet-bot Innerloop Windows_NT Release Build and Test

**Comment by JeremyKuhne** [comment]
@dotnet-bot test Innerloop Windows_NT Release Build and Test

**Comment by JeremyKuhne** [comment]
@dotnet-bot test Innerloop CentOS7.1 Debug Build and Test

**Comment by JeremyKuhne** [comment]
@dotnet-bot test Innerloop CentOS7.1 Debug Build and Test

**Pull request merged**